conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -9,11 +9,37 @@
 from fastapi import APIRouter, Body, Depends, HTTPException, Query, Request
 from fastapi.responses import PlainTextResponse, RedirectResponse
 
-router_api = APIRouter(
```

Comment:
Renamed to `router_api_v1`. Based on the proposed [BC policy](https://github.com/conda-incubator/conda-store/issues/595#issuecomment-1783843666), I shouldn't have touched this, but I think there's zero chance this can be used by anyone except this application, since it's only for defining routes. So I renamed it to make it easier to see which API version is used.